app/extensions/brave/content/scripts/blockCanvasFingerprinting.js

```diff
@@ -267,4 +279,53 @@ if (chrome.contentSettings.canvasFingerprinting == 'block') {
     type: 'WebRTC',
     methodName: 'navigator.mediaDevices.enumerateDevices'
   })
+
+  var propertiesToReport = new Set(canvasMethods
+    .concat(canvasElementMethods)
+    .concat(webglMethods)
+    .concat(webglMethods)
```

Comment:
webgl is concat'ed twice

Reply:
Fixed in 

Sorry for all these issues, most just curious if the general approach is appealing for a patch.  If so, and if fadd50814 is not up to snuff, I'll give the PR a careful head-to-toe before sending it back to you